apps/meteor/server/oauth2-server/oauth.ts

```diff
@@ -21,6 +21,17 @@ export class OAuth2Server {
 
 		this.config = config;
 		this.app = express();
+		this.app.use(
+			'/oauth/*',
+			express.json({
+				limit: '50mb',
+			}),
+			express.urlencoded({
+				extended: true,
+				limit: '50mb',
+			}),
```

Comment:
same questions about limit or was it already this as default now needing explicit mention

Reply:
since we removed from the 'root' route we just set for each remaining routes